mica/archive/asp_l1.py

```diff
@@ -147,6 +150,48 @@ def get_files(obsid=None, start=None, stop=None,
                              revision=revision, content=content)
 
 
+def get_atts(obsid=None, start=None, stop=None, revision=None):
+    """
+    Get the ground aspect solution quaternions and times covering obsid or start to stop,
+    in the ACA frame.
+
+    :obsid: obsid
+    :start: start time (DateTime compat)
+    :stop: stop time (DateTime compat)
+    :revision: aspect pipeline processing revision (integer version, None, or 'last')
+
+    :returns: Nx4 np.array of quaternions, np.array of N times, list of dict with header from each asol file.
+    """
+    if revision == 'all':
+        raise ValueError("revision 'all' doesn't really make sense for this function")
+    asol_files = sorted(get_files(obsid=obsid, start=start, stop=stop,
```

Comment:
For the general/time case, these need to be sorted by time and not by file name (which has the obsid in the path).